fec.config.js

```diff
@@ -12,7 +13,23 @@ module.exports = {
   devtool: 'hidden-source-map',
   useProxy: true,
   interceptChromeConfig: false,
-  plugins: [],
+  plugins: [
+    ...(process.env.SENTRY_AUTH_TOKEN
+      ? [
+          sentryWebpackPlugin({
+            authToken: process.env.SENTRY_AUTH_TOKEN,
+            org: process.env.SENTRY_ORG,
+            project: process.env.SENTRY_PROJECT,
+            _experiments: {
```

Comment:
This no longer needs to be under experiments, you can mock the moduleMetadata section from remediations: https://github.com/RedHatInsights/insights-remediations-frontend/blob/2fa843a88cc3524aec237ab1927394ac02d3b7e4/fec.config.js#L29. 

If you wanted to make the DSN a secret, there way be some additional work to be completed in the shared builder file. As of now im not sure how konflux would access the variable like this